components/schemes/Container/BUILD

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018-present The Material Components for iOS Authors. All
+# Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//:material_components_ios.bzl",
+     "mdc_objc_library",
+     "mdc_public_objc_library",
+     "mdc_unit_test_suite")
+
+licenses(["notice"])  # Apache 2.0
+
+mdc_public_objc_library(
+    name = "Container",
+    deps = [
+        "//components/schemes/Color",
+        "//components/schemes/Shape",
+        "//components/schemes/Typography",
+    ],
+)
+
+mdc_objc_library(
+    name = "unit_test_sources",
+    testonly = 1,
+    srcs = glob(["tests/unit/*.m"]),
+    hdrs = glob(["tests/unit/*.h"]),
+    includes = ["src"],
```

Comment:
I think the `includes` can be removed since all of the included headers will be available via the umbrella header.

Reply:
Good to know. I didn't know how umbrella header is handled in the kokoro ci process.